src/partials/support-status-banner/banner-for-eol-scheduled.hbs

```diff
@@ -0,0 +1,18 @@
+<div class="notice-banner">
+    <button class="notice-banner-close-button" title="Close notice banner">&times;</button>
+    <p>
+        {{#if page.latest.url}}
+        This version of the product
+        {{else}}
+        This product
+        {{/if}}
+        is scheduled for
+        <a class="notice-banner-link" href="https://www.mulesoft.com/legal/versioning-back-support-policy"
+            target="_blank" rel="noopener noreferrer">End of Life.</a>
```

Comment:
@hannanelson you can look at the wording of all the banner-for-* files starting from this one.